setup.py

```diff
@@ -11,7 +11,7 @@
 
 general_requirements = [
     "pytest>=8,<9",
-    "numpy<1.25",
+    "numpy==1.26.4",
```

Comment:
does this need to be so specific, or could it be a range or single bound?

Reply:
It could probably be a range, but the broader problem is that there are no versions of NumPy that support both Python 3.12 and Python 3.7. So if I say "at least 1.26.0," so it works for 3.12, then it doesn't work for anything before 3.9.